packages/subgraph/src/utils.ts

```diff
@@ -258,6 +258,18 @@ export async function updateSpaceCachedMetrics(
     console.log(`Updated ${eventType} rolling window metrics for space ${spaceId}`)
 }
 
+export function calculateWeightedRating(averageRating: number, reviewCount: number): number {
+    // Bayesian average formula: pulls rating toward global average when few reviews exist
+    // This prevents spaces with 1-2 five-star reviews from ranking highest
+    const globalAverage = 3.0 // Middle rating on 1-5 scale
+    const minReviews = 10 // Threshold for confidence
+
+    const weightedRating =
+        (reviewCount * averageRating + minReviews * globalAverage) / (reviewCount + minReviews)
```

Comment:
idk this is kinda lame if my town has 100 5 star reviews it's going to render as 4.81?